Source/WebGPU/WebGPU/RenderPassEncoder.mm

```diff
@@ -99,17 +102,26 @@
 
 void RenderPassEncoder::drawIndexed(uint32_t indexCount, uint32_t instanceCount, uint32_t firstIndex, int32_t baseVertex, uint32_t firstInstance)
 {
+    if (!m_renderPipeline)
+        return;
```

Comment:
Oh, one more thing - If this occurs, I don't believe we're supposed to just silently drop _just the draw call_ but instead I believe we're supposed to make the whole command buffer as erroneous.

Reply:
Oh yes the spec says to make RenderPassenEncoder invalid. I should do that too